entity-framework/ef6/building-an-mvc-3-app-with-model-first-and-entity-framework-4-12.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
this topic should probably be deleted

Reply:
Agreed. I have deleted a bunch of empty pages and dead links, but not from the tic yet. I will add EF6/ and see what is left.